### ISSUE TITLE
Very slow download speed for Instagram [Broken]

### ISSUE THREAD

**Issue by ahm750** [yt-dlp/yt-dlp#718]
## Checklist

- [x] I'm reporting a broken site support
- [x] I've verified that I'm running yt-dlp version **2021.08.10**
- [x] I've checked that all provided URLs are alive and playable in a browser
- [x] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [x] I've searched the bugtracker for similar issues including closed ones

## Verbose log

```
[debug] Encodings: locale UTF-8, fs utf-8, out utf-8, pref UTF-8
[debug] yt-dlp version 2021.08.10 (zip)
[debug] Python version 3.9.5 (CPython 64bit) - macOS-10.15.3-x86_64-i386-64bit
[debug] exe versions: ffmpeg 4.4, ffprobe 4.4, rtmpdump 2.4
[debug] Optional libraries: sqlite
[debug] Proxy map: {}
[debug] [Instagram] Extracting URL: https://www.instagram.com/reel/CR2c5qMITkg/?utm_source=ig_web_copy_link
[Instagram] CR2c5qMITkg: Downloading webpage
[debug] Default format spec: bestvideo*+bestaudio/best
[info] CR2c5qMITkg: Downloading 1 format(s): 0
[debug] Invoking downloader on "https://scontent-ort2-2.cdninstagram.com/v/t50.2886-16/10000000_580278303381812_1507946641509866431_n.mp4?_nc_ht=scontent-ort2-2.cdninstagram.com&_nc_cat=108&_nc_ohc=2pSsWBjOWVAAX-wFU9v&edm=APfKNqwBAAAA&ccb=7-4&oe=611D2B5F&oh=d45ea14d787aab43ce0c09ea044e237e&_nc_sid=74f7ba"
[download] Resuming download at byte 387232
[download] Destination: Video by surcuito.ccs [CR2c5qMITkg].mp4
[download]   3.6% of 18.55MiB at 43.29KiB/s ETA 07:03

```

## Description

Downloads from Instagram are super slow, about 50kb/sec on average. I have a decent internet connection (100MB) and other websites download quickly, but the download speed is terribly slow for Instagram. Anyone knows why?

**Comment by pukkandan** [comment]
Fixed in 3dd39c5f9a55b8169cf35bc2b33a08842b3c40fe

**Comment by pukkandan** [comment]
Since the original issue as once patched, better to open a new report
